code/game/machinery/hologram.dm

```diff
@@ -90,6 +97,14 @@ var/const/HOLOPAD_MODE = RANGE_BASED
 				return
 			if(last_request + 200 < world.time) //don't spam other people with requests either, you jerk!
 				last_request = world.time
+				var/obj/item/card/id/I = user.GetIdCard()
+				if(!user.GetIdCard())
+					to_chat(user, "<span class='notice'>You need authorization to use the holocall system. please equip a valid ID card.</span>")

```

Comment:
should try to use span("notice", "text.") where you can in these changes instead of span class.